composer/callbacks/image_visualizer.py

```diff
@@ -171,32 +149,26 @@ def eval_after_forward(self, state: State, logger: Logger):
 
 
 def _make_input_images(inputs: torch.Tensor, num_images: int, channels_last: bool = False):
```

Comment:
seems like the `channels_last` arg is no longer needed here?

Reply:
done